src/features/auth/actions/systemMessage.ts

```diff
@@ -0,0 +1,18 @@
+export async function getSystemMessage(): Promise<string | null> {
+  const response = await window.fetch(
+    `https://sunflower-land.com/system-message.txt`,
```

Comment:
`https://sunflower-land.com/system-message.txt?v=${CONFIG.RELEASE_VERSION}`,

This one might be a good balance between caching, and busting the cache on FE update